components/controls/GlobalMessages.tsx

```diff
@@ -60,14 +60,14 @@ export class GlobalMessage extends React.PureComponent<{
     const { message } = this.props;
     return (
       <div
-        className={classNames('process-spinner', 'shown', {
-          'process-spinner-failed': message.level === 'ERROR',
-          'process-spinner-success': message.level === 'SUCCESS'
+        className={classNames('process-message', 'shown', {
+          'process-message-failed': message.level === 'ERROR',
+          'process-message-success': message.level === 'SUCCESS'
```

Comment:
Why not make it a styled-component?

Reply:
Ahhhh right! We can make styled-component! I had forgotten!
I'll have a look to transform it!